launcher_scripts/conf/training/gpt3/40b.yaml

```diff
@@ -108,14 +108,16 @@ model:
   grad_allreduce_chunk_size_mb: 125
 
   ## Transformer Engine
+  # To use fp8, change only `fp8` to `True`.
+  # The rest of fp8 knobs are set for the fp8 training mode, which are ignored in non-fp8 training
```

Comment:
update comment to include # To use fp8, transformer_engine=True and fp8=True

Reply:
Done